app/controllers/application_controller.rb

```diff
@@ -2,4 +2,40 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  skip_before_filter :verify_authenticity_token
+
+
+  #query ups for prices
+  def get_rates
+    origin = ActiveShipping::Location.new(country: 'US',
+                                           state: 'CA',
+                                           city: 'Beverly Hills',
+                                           zip: '90210')
```

Comment:
I like how you've split this up onto separate lines. It makes it easier to read.